examples/test-app/views/question-globals.njk

```diff
@@ -0,0 +1,13 @@
+{% block proposition_header %}
+  {% from "look-and-feel/components/nav.njk" import propositionalNavigation %}
+  {{ propositionalNavigation('Apply for Divorce') }}
+{% endblock %}
+
+{% block header_class %}
+  with-proposition
+{% endblock %}
+
+{% block head -%}
+<link href="{{ asset_path }}main.css" media="screen" rel="stylesheet" />
+<script type="text/javascript" src="{{ asset_path }}main.js"></script>
```

Comment:
Is it correct in assuming the asset path will contain the ending `/`

Reply:
yeah, I kinda regret that.

Tempted to make a function in look-and-feel like rails has: `{{ asset('main.css') }}` which resolves the filepath for you. and then people will stop asking this question :wink: .